### ISSUE TITLE
Round buttons in alert

### ISSUE THREAD

**Issue by sergeygarazha** [huri000/SwiftEntryKit#117]
**Describe the bug**
Hello! Thank you for great library!
When I'm using alert with rounded corners, bottom corner of buttons not rounded.

**To Reproduce**
Here is a code:
```swift
// attributes
var attributes = EKAttributes.centerFloat
attributes.displayDuration = .infinity
attributes.windowLevel = .alerts
attributes.entryBackground = .visualEffect(style: .dark)
attributes.screenInteraction = .dismiss
attributes.hapticFeedbackType = .warning
attributes.position = .center
attributes.roundCorners = .all(radius: 12)
attributes.border = .value(color: .darkGray, width: 2)
attributes.screenBackground = .color(color: .black.withAlphaComponent(0.45))
attributes.positionConstraints.maxSize = EKAttributes.PositionConstraints.Size(width: EKAttributes.PositionConstraints.Edge.offset(value: 65), height: EKAttributes.PositionConstraints.Edge.intrinsic)
attributes.shadow = .active(with: EKAttributes.Shadow.Value(color: .black, opacity: 0.7, radius: 15, offset: .zero))
// content
let messageImage = EKProperty.ImageContent(imageName: "alert")
let titleLabelStyle = EKProperty.LabelStyle(font: UIFont.bold(20), color: .white, alignment: NSTextAlignment.center, numberOfLines: 0)
let titleLabelContent = EKProperty.LabelContent(text: title, style: titleLabelStyle)
let descriptionLabelStyle = EKProperty.LabelStyle(font: UIFont.systemFont(ofSize: 15), color: .white, alignment: NSTextAlignment.center, numberOfLines: 0)
let descriptionLabelContent = EKProperty.LabelContent(text: message, style: descriptionLabelStyle)
let simpleMessage = EKSimpleMessage(image: messageImage, title: titleLabelContent, description: descriptionLabelContent)
// buttons
let firstButton = configureButton(fontSize: 20, titleColor: .red, title: firstTitle, backgroundColor: .white, highlightedColor: .white, action: firstAction)
let secondButton = configureButton(fontSize: 20, titleColor: .blue, title: secondTitle, backgroundColor: .white, highlightedColor: .white, action: secondAction)
let buttonsBar = EKProperty.ButtonBarContent(with: firstButton, secondButton, separatorColor: .lightGray, expandAnimatedly: true)
// presenting
let alertMessage = EKAlertMessage(simpleMessage: message, imagePosition: .top, buttonBarContent: buttons)
let view = EKAlertMessageView(with: alertMessage)
SwiftEntryKit.display(entry: view, using: attributes)
```

As I understand, layer mask not applied to buttons layers:
<img width="320" alt="image" src="https://user-images.githubusercontent.com/1624643/46280684-3375be00-c575-11e8-916f-4b1aa1f56322.png">
<img width="376" alt="image" src="https://user-images.githubusercontent.com/1624643/46281491-88b2cf00-c577-11e8-8429-58ff9dc95a34.png">

**Comment by huri000** [comment]
Hi @sergeygarazha! Thank for opening the issue. Can you elaborate a bit more? Xcode version? SwiftEntryKit version? Devices (iOS, Simulator)?

**Comment by sergeygarazha** [comment]
Sure! 
- Xcode 10.0 (10A255)
- SwiftEntryKit (0.7.2):
- QuickLayout (= 2.0.2)
iPhoneX (iOS 12.0), iPhone 8 simulator (iOS 11.4)

**Comment by Sndrsn** [comment]
I've run into this as well. I didn't notice it at first, because most of my buttons have a clear background. But as soon as I started trying to use a highlight color for the buttons I noticed it.
Xcode 10.0
SwiftEntryKit 0.8.2
iPhone XR Simulator

**Comment by huri000** [comment]
A fix is available now - release `0.8.7` 🎉

The adjustment was made specifically for `EKAlertMessageView` and `EKRatingMessageView` using a unique protocol that is responsible for the layout of the bottom part of those views. 
You should be able to set whatever background color you want for the button bar.

@sergeygarazha, @Sndrsn, Please let me know if that works for you.
